mod-orders-storage/schemas/eresource.json

```diff
@@ -12,8 +12,15 @@
       "type": "integer"
     },
     "createInventory": {
-      "description": "whether or not to create an inventory record for this resource",
-      "type": "boolean"
+      "description": "Shows what inventory objects need to be created for electronic resource",
+      "type": "string",
+      "enum": [
+        "INSTANCE_HOLDING_ITEM",
+        "INSTANCE_HOLDING",
+        "INSTANCE",
+        "NONE"
+      ],
+      "default": "INSTANCE_HOLDING"
```

Comment:
Please remove default. As described in [MODORDERS-178](https://issues.folio.org/browse/MODORDERS-178):
> NOTE: The defaults identified above should NOT be set in the schemas, but rather in the code. There are a few reasons for this... We plan on later implementing the ability for tenants to specify defaults which override the system-wide defaults noted in this story. Another reason relates to orders of type "Other" and is described below.

Reply:
Can we name these enum values something like:
"Instance, holding, item"
"Instance, holding"
"Instance"
"None"

instead of the ALL_CAPS_UNDERSCORE_DELIM values?

Reply:
> Can we name these enum values something like:
> `"Instance, holding, item"`
> `"Instance, holding"`
> `"Instance"`
> `"None"`
> 
> instead of the `ALL_CAPS_UNDERSCORE_DELIM` values?

Yes. I discussed it with Aliaksey again and it looks like it's no matter for the UI how the enum values will be named. So I change naming as it described in jira.